Protos/BazelRemoteAPI/googleapis/google/rpc/error_details.proto

```diff
@@ -89,36 +89,39 @@ message QuotaFailure {
 //
 // Example of an error when contacting the "pubsub.googleapis.com" API when it
 // is not enabled:
+//
 //     { "reason": "API_DISABLED"
 //       "domain": "googleapis.com"
 //       "metadata": {
 //         "resource": "projects/123",
 //         "service": "pubsub.googleapis.com"
 //       }
 //     }
+//
 // This response indicates that the pubsub.googleapis.com API is not enabled.
 //
 // Example of an error that is returned when attempting to create a Spanner
 // instance in a region that is out of stock:
+//
 //     { "reason": "STOCKOUT"
 //       "domain": "spanner.googleapis.com",
 //       "metadata": {
 //         "availableRegions": "us-central1,us-east2"
 //       }
 //     }
-//
 message ErrorInfo {
   // The reason of the error. This is a constant value that identifies the
   // proximate cause of the error. Error reasons are unique within a particular
   // domain of errors. This should be at most 63 characters and match
   // /[A-Z0-9_]+/.
   string reason = 1;
 
-  // The logical grouping to which the "reason" belongs.  Often "domain" will
-  // contain the registered service name of the tool or product that is the
-  // source of the error. Example: "pubsub.googleapis.com". If the error is
-  // common across many APIs, the first segment of the example above will be
-  // omitted.  The value will be, "googleapis.com".
+  // The logical grouping to which the "reason" belongs. The error domain
```

Comment:
these changes appear to be unrelated to the PR, could you remove them to keep the PR focused on the changes?

Reply:
I didn't change it, it is what `make generate` did. Do you ask me to change it manually?

Reply:
Ah, I think I forgot to rebase

Reply:
Changed manually